src/app/shared/services/chat/offline/chat.flow.ts

```diff
@@ -0,0 +1,207 @@
+import { Observable, of, BehaviorSubject } from 'rxjs';
+import { ChatMessage, ChatResponseOption } from '../chat-msg.model';
+import { FlowStatusChange } from './offline-chat.service';
+import { RapidProFlowExport } from './rapid-pro-export.model';
+
+export interface ChatFlow {
```

Comment:
Exporting an interface from the class could get messy if used elsewhere (pretty easy to end up with circular dependencies), so might be better to have in the other typings/models files